FirebasePerformance/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Version 8.2.0
 * Update log messages with proper log levels.
+* Fix empty values in `network_info.request_completed_time_us` field from the [data schema](https://firebase.google.com/docs/perf-mon/bigquery-export#detailed_data_schema).
```

Comment:
While adding the `Completed` implementation, it seems to be removing the `Received` implementation. Does that need to be noted in the Release Notes?

Reply:
I removed `Received` implementation from `didSendBodyData:totalBytesWritten:totalBytesExpectedToWrite:` because the response is not actually received at that point. `Received` is still captured in other places. For this reason, I think I don't have to update the Release Notes.